src/main/java/util/RandomFieldPopulator.java

```diff
@@ -0,0 +1,13 @@
+package util;
+
+import com.github.javafaker.Faker;
+
+public class RandomFieldPopulator {
```

Comment:
it is not necessary to use separate class for this one method. Move the method to the TestUtils class and remove this class

Reply:
Fixed